common/workunit/workunit.cpp

```diff
@@ -923,6 +923,7 @@ class CDaliWorkUnit : public CPersistedWorkUnit
             if (progress)
             {
                 StringBuffer path;
+                // MORE I don't think this works! Gavin changed the format... Jake?
```

Comment:
@jakesmith Looks like the node state is stored in a slightly unexpected place. If it's like that in 5.x too we may have to leave it